### DISCUSSION TITLE
Is there any rewrite version?

### DISCUSSION THREAD

**Discussion by Placeblock** [Rapptz/discord.py#7086] [Q&A] [answered]
Hey Users,

today i just read about the "rewrite" version of discordpy and i wanted to test it myself :D

I tried to install it by executing this: pip install -U git+https://github.com/Rapptz/discord.py@rewrite
but there was the error : ddod not find branch or tag "rewrite"...

has the rewrite version merged into master?

maybe you can help me, how to install rewrite version : )

Have a great day 👍

**Comment by LostLuma** [comment, marked as answer]
The `rewrite` branch was used when the library was being re-written for the 1.0 release, and has indeed become the `master` branch.

You should generally not install from the `master` branch as version 2.0 is being actively worked on, and there will be breaking changes without official notice.

**Reply by Placeblock** [reply]
Thanks for your fast response!
But how should i install discord.py then?
Because when i installed it like so : pip install discord.py
there was at example no member.voice_channel attribute
(example: https://stackoverflow.com/questions/45770016/how-do-i-make-my-discord-bot-play-audio-from-youtube)

👍

**Reply by wasi-master** [reply]
> Thanks for your fast response!
> But how should i install discord.py then?
> Because when i installed it like so : pip install discord.py
> there was at example no member.voice_channel attribute
> (example: https://stackoverflow.com/questions/45770016/how-do-i-make-my-discord-bot-play-audio-from-youtube)
> 
> 👍 

member.voice_channel no longer exists, Use `member.voice.channel` instead
Note that member.voice will be None if member isn’t in a voice channel, so be careful about that
